contracts/main/price-oracles/lib/FathomSwapLibrary.sol

```diff
@@ -11,49 +11,9 @@ library FathomSwapLibrary {
         require(token0 != address(0), "FathomSwapLibrary: ZERO_ADDRESS");
     }
 
-    function pairFor(address factory, address tokenA, address tokenB) internal view returns (address pair) {
-        return IFathomSwapFactory(factory).getPair(tokenA, tokenB); // For easy testing
-    }
-
-    function getReserves(address factory, address tokenA, address tokenB) internal view returns (uint reserveA, uint reserveB) {
```

Comment:
2.2.7 There is no validation of return data in
FathomSwapLibrary

audit fix.

Instead of staticCall, interface will be used in DexPriceOracle.sol and SlidingWindowDexOracle.sol